src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/CSharpSyntaxGenerator.csproj

```diff
@@ -5,22 +5,24 @@
     <Platform Condition="'$(Platform)' == ''">x64</Platform>
     <PlatformTarget>x64</PlatformTarget>
     <Platforms>x64</Platforms>
-    <OutputType>Exe</OutputType>
-    <RootNamespace>Roslyn.Compilers.CSharp.Internal.CSharpSyntaxGenerator</RootNamespace>
+    <RootNamespace>CSharpSyntaxGenerator</RootNamespace>
```

Comment:
All of the code in this project was in CSharpSyntaxGenerator; I don't know why the RootNamespace was what it was originally set to.